### DISCUSSION TITLE
CLI client for sending encrypted email?

### DISCUSSION THREAD

**Discussion by dominic-p** [tutao/tutanota#3397] [Q&A]
I'm a new Tutanota user, and so far I'm really enjoying it. Since PGP is not currently supported (#198), I'm struggling to find a way to send an encrypted email via the command line. My use case is for server status reports and notifications. This kind of information needs to be sent automatically and can be sensitive, so it would be nice if it could be encrypted.

Since the code base is primarily JS, would it be possible to make a thin, send-only Node CLI?

For reference, this same issue was [discussed here](https://www.reddit.com/r/commandline/comments/88omb2/looking_for_a_command_line_app_to_access_tutanota/), but it doesn't look like there was a great solution mentioned there.

**Comment by charlag** [comment]
Hi
Someone can have a stab at it. It will be easier once we modularize better but you can hack your way through already.

Automated mails is not our primary use case though and you might be better off looking into another solution.

**Reply by dominic-p** [reply]
Thanks for the response. If I get some free time, I might take a crack at it. Otherwise, I appreciate you letting me know where the priorities are.
